.code-samples.meilisearch.yaml

```diff
@@ -244,16 +244,18 @@ documents_guide_add_movie_1: |-
   await client.index('movies').addDocuments([
     {'movie_id': '123sq178', 'title': 'Amélie Poulain'}
   ]);
-document_guide_create_index_primary_key: |-
-  await client.createIndex('movies', primaryKey: 'reference_number');
-document_guide_add_document_primary_key: |-
+primary_field_guide_create_index_primary_key: |-
+  await client.createIndex('books', primaryKey: 'reference_number');
+primary_field_guide_update_document_primary_key: |-
+  await client.updateIndex('books', 'title');
+primary_field_guide_add_document_primary_key: |-
   await client.index('movies').addDocuments([
     {
       reference_number: 287947,
-      title: 'Shazam',
-      poster: 'https://image.tmdb.org/t/p/w1280/xnopI5Xtky18MPhK40cZAGAOVeV.jpg',
-      overview: 'A boy is given the ability to become an adult superhero in times of need with a single magic word.',
-      release_date: '2019-03-23'
+      title: 'Diary of a Wimpy Kid',
+      author: 'Jeff Kinney',
+      genres: ['comedy', 'humor'],
+      price: 5.00
     }
   ], primaryKey: 'reference_number')
```

Comment:
Why this one does not have a `;` at the end of the line like the others? It's on purpose?

Reply:
Nope, the semicolon is required! I'll check all the samples and add it on them thanks!